doc/source/basics.rst

```diff
@@ -1726,8 +1726,9 @@ Sorting
    The sorting API is substantially changed in 0.17.0, see :ref:`here <whatsnew_0170.api_breaking.sorting>` for these changes.
    In particular, all sorting methods now return a new object by default, and **DO NOT** operate in-place (except by passing ``inplace=True``).
 
-There are two obvious kinds of sorting that you may be interested in: sorting
-by label and sorting by actual values.
+There are three obvious kinds of sorting that you may be interested in: sorting
```

Comment:
I know this word was in the original doc, but I'm a little uneasy about judgmental words like "obvious."  I think we can just remove it.

Reply:
Agreed, I've reworded it.